cypress.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'cypress';
+import * as plugins from './test/plugins/index.js';
+
+export default defineConfig({
+    defaultCommandTimeout: 15000,
+    fileServerFolder: 'test',
+    fixturesFolder: 'test/fixtures',
+    screenshotsFolder: 'test/screenshots',
+    video: false,
+    videosFolder: 'test/videos',
+    viewportHeight: 1260,
+    viewportWidth: 1600,
+    e2e: {
+        // We've imported your old cypress plugins here.
```

Comment:
Should this comment be removed now or are we still doing something weird?

Reply:
Oh yeah, we can remove it now; I'll update that